mesonbuild/modules/pkgconfig.py

```diff
@@ -329,6 +330,27 @@ def _generate_pkgconfig_file(self, state, deps, subdirs, name, description,
                                  url, version, pcfile, conflicts, variables,
                                  unescaped_variables, uninstalled=False, dataonly=False):
         coredata = state.environment.get_coredata()
+        referenced_vars = {'prefix'}
+        optnames = [x.name for x in BUILTIN_DIR_OPTIONS.keys()]
+
+        if not dataonly:
+            # includedir is always implied, although libdir may not be
+            # needed for header-only libraries
+            referenced_vars |= {'includedir'}
```

Comment:
We can always add `prefix` in `referenced_vars` too.

Reply:
It's initialized with that, because we almost always need to include it. Any other variable is bound to refer to it, and explicitly including it might get a teensy bit hairy.

Reply:
Oh what the heck, I included it. :)